src/components/Filters/Filters.enums.ts

```diff
@@ -0,0 +1,10 @@
+export const InputTypes = {
+  string: 'string',
+  multipleString: 'multipleString',
+  uint: 'uint',
```

Comment:
I would rename it to `count`, no need to bring here terms like `unsigned integer` :D